creator-node/test/issueSyncRequest.jobProcessor.test.js

```diff
@@ -36,6 +36,7 @@ describe('test issueSyncRequest job processor', function () {
     syncRequestParameters
 
   beforeEach(async function () {
+    config.set('entityManagerReplicaSetEnabled', true)
```

Comment:
intentional to test this?

Reply:
yeah I wanted it to reach the code path with using entity manager. I didn't add duplicate tests for the ursm code path because the tests passed before, it's 90% the same, and it'll be removed soon enough